en/deploy-tidb-monitor-across-multiple-kubernetes.md

```diff
@@ -0,0 +1,328 @@
+    cluster_namespace="pingcap"
+    kubernetes_cluster_name="kind-cluster-1"
+    storageclass_name="local-storage"
+    ```
+
+2. Create a `TidbMonitor` component by running the following command:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    cat << EOF | kubectl apply -n ${cluster_namespace} -f -
+    apiVersion: pingcap.com/v1alpha1
+    kind: TidbMonitor
+    metadata:
+      name: ${cluster_name}
+    spec:
+      clusters:
+      - name: ${cluster_name}
+        namespace: ${cluster_namespace}
+      externalLabels:
+        # k8s_clsuter indicates the k8s cluster name, you can change
+        # the label's name on your own, but you should notice that the
+        # "cluster" label has been used by the TiDB metrics already.
+        # For more information, please refer to the issue
+        # https://github.com/pingcap/tidb-operator/issues/4219.
+        k8s_clsuter: ${kubernetes_cluster_name}
+        # add other meta labels here
+        #region: us-east-1
+      initializer:
+        baseImage: pingcap/tidb-monitor-initializer
+        version: v5.4.0
+      persistent: true
+      storage: 20Gi
+      storageClassName: ${storageclass_name}
+      prometheus:
+        baseImage: prom/prometheus
+        logLevel: info
+        version: v2.27.1
+      reloader:
+        baseImage: pingcap/tidb-monitor-reloader
+        version: v1.0.1
+      imagePullPolicy: IfNotPresent
+    EOF
+    ```
+
+#### Configure Federation Prometheus
+
+For details on Federation, refer to [Federation](https://prometheus.io/docs/prometheus/latest/federation/#hierarchical-federation). After the deployment of Prometheus, you need to modify the Prometheus configuration, and add the host information of the Prometheus (`TidbMonitor`) to be aggregated.
+
+```yaml
+scrape_configs:
+  - job_name: 'federate'
+    scrape_interval: 15s
+
+    honor_labels: true
+    metrics_path: '/federate'
+
+    params:
+      'match[]':
+        - '{__name__=~".+"}'
+
+    static_configs:
+      - targets:
+        - 'source-prometheus-1:9090'
+        - 'source-prometheus-2:9090'
+        - 'source-prometheus-3:9090'
+```
+
+</div>
+</SimpleTab>
+
+## Visualize monitoring data using Grafana
+
+After collecting data using Prometheus, you can visualize multi-cluster monitoring data using Grafana.
+
+1. Obtain the Grafana Dashboards related to TiDB components by running the following command:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    # set tidb version here
+    version=nightly
+    docker run --rm -i -v ${PWD}/dashboards:/dashboards/ pingcap/tidb-monitor-initializer:${version} && \
+    cd dashboards
+    ```
+
+    > **Note:**
+    >
+    > In the command above, `${version}` is the version of the Initializer image, which should be consistent with the TiDB version. However, currently only the `nightly` version of the Initializer image supports **multiple Kubernetes clusters** monitoring, so this example uses the `nightly` image.
```

Comment:
since TiDB 6.0 has been released, maybe change the example to v6.0.0 here?

Reply:
Sure. It's your call.

Reply:
Updated in [5b7f056](https://github.com/pingcap/docs-tidb-operator/pull/1775/commits/5b7f0568c0c5ff373ae35cb15fd08b77843db841)